tests/unittest/bindings/test_executor_bindings.py

```
@@ -102,6 +102,31 @@ def test_executor_with_managed_weights(model_files, model_path):
     assert executor.can_enqueue_requests() == True
 
 
+def test_executor_with_torch_managed_weights(model_files, model_path):
```

Ruff (0.15.5)

[warning] Unused function argument: `model_files`

(ARG001)